source/conf.py

```diff
@@ -46,7 +46,6 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx.ext.todo',
-    'sphinx_copybutton',
```

Comment:
why are you removing copybutton?

Reply:
Accident. I hadn't bothered installing that so I always kept deleting it. But a couple of days ago I installed it so shouldn't have such problems anymore.